contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-23
+title: You can now organize your PostHog data
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
```

Comment:
Perhaps we could expand a little more on the whole 'data explosion' theme and its impacts, why we wanted to solve it?

Reply:
Avoid PH acronym?